numpy/core/fromnumeric.py

```diff
@@ -1361,8 +1361,9 @@ def ravel(a, order='C'):
 
     Returns
     -------
-    1d_array : ndarray
-        Output of the same dtype as `a`, and of shape ``(a.size,)``.
+    y : array_like
+        Array of the same type as `a`, and of shape ``(a.size,)``
+        (or ``(1, a.size)`` for matrices).
```

Comment:
@charris, I've added a note in the docstring about the update in v1.10.  Can you check it to make sure that's what you meant?